boards/arm/stm32373c_eval/doc/stm32373c_eval.rst

```diff
@@ -118,9 +118,7 @@ Programming and Debugging
 Flashing
 ========
 
-STM32373C-EVAL board includes an ST-LINK/V2-1 embedded debug tool interface.
-At power-on, the board is in firmware-upgrade mode (also called DFU for
-"Device Firmware Upgrade"), allowing the firmware to be updated through the USB.
+STM32F411E-DISCO Discovery kit includes an ST-LINK/V2 embedded debug tool interface.
```

Comment:
Wrong board name I guess

Reply:
Sure! my bad... copy/paste crap! updated!. Anyway, if it depends on board revisions, better to add a config option as you point. (out of my reach atm, I'm new with Zephyr!)